src/components/Button.tsx

```diff
@@ -41,7 +41,9 @@ const Button: React.FC<ButtonProps> = ({
     warning: "bg-warning text-black hover:bg-secondary hover:text-white",
   };
 
-  const disabledClasses = disabled ? "opacity-60 cursor-not-allowed" : "";
+  const disabledClasses = disabled
+    ? "opacity-60 cursor-not-allowed pointer-events-none"
+    : "";
 
   return (
     <a
```

Comment:
important: you need to find a way to block/stop the click event in this `a` element.

Reply:
@inderps @admy7 what is the easiest way to do that in React?

Reply:
can you check the change now?

Reply:
@EmiPali thanks for your prompt fix.

Although it may work, it is not a clean solution. I am not used to React, but in Angular the clickable/actionable components or elements used to have the attribute `disabled`, like `<a className="..." disabled ...>`. Perhaps you have it or similar kind of boolean attribute for preventing users to trigger the click event.

Would you mind checking once again? 

If you find a cleaner solution, perhaps you can simplify the CSS class your are using. Again in Angular, `disabled` used to come with specific CSS effects that changes opacity, cursor and other things to also visualy disable the link or button.